### PR TITLE
Updating Readme

### PR DESCRIPTION
**Pull request by astoycos**
- Add a background section with some extra resources 
- Add ability to use Openshift 4.x GUI
- Update other small parts of ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,92 @@
-# Example setup the IoT simulator
+# Iot Simulator
```
Comment (ctron) [inline README.md]:
```suggestion
# IoT Simulator
```

File: README.md [hunk 2]
```diff
@@ -1,40 +1,92 @@
...
+![EnMasse Overview](https://raw.githubusercontent.com/astoycos/iot-simulator-example/master/doc/enmasse_iot_view.png)
+
+## The setup 
+
+This demostration is split up into 4 major modules that are deployed in 4 different pods during use 
+
+1. Simulator
+    * Manages the Producer and Consumer
+2. Producer 
+    * Generates arbitrary MQTT or HTPP telemetry messages that are sent to EnMasse's IOT protocol adapter(Eclipse Hono)
```
Comment (ctron) [inline README.md]:
```suggestion
    * Generates arbitrary MQTT or HTTP telemetry messages that are sent to EnMasse's IoT protocol adapter (Eclipse Hono)
```

File: README.md [hunk 3]
```diff
@@ -1,40 +1,92 @@
...
-    mkdir enmasse
-    cd enmasse
-    tar --strip-components=1 -xzf ../enmasse.tar.gz
-    cd ..
 
-## Using CRC
+# Deploy the IoT simulator
 
-When using CRC, create a new machine using the following command:
+This Demo can be deployed on either a **remote** or **local** openshift instance
```
Comment (ctron) [inline README.md]:
```suggestion
This Demo can be deployed on either a **remote** or **local** OpenShift instance 
```

File: README.md [hunk 4]
```diff
@@ -1,40 +1,92 @@
...
 
     crc start
 
 ## Prepare your OpenShift cluster
 
-Create a new project:
+1. Log in as admin
+2. Make new project titled "enmasse-infra" 
+    
+    `oc new-project enmasse-infra`
```
Comment (ctron) [inline README.md]:
```suggestion
       oc new-project enmasse-infra
```

File: README.md [hunk 5]
```diff
@@ -1,40 +1,92 @@
...
+
+Unpack the archive:
+
+    mkdir enmasse
+    cd enmasse
+    tar --strip-components=1 -xzf ../enmasse.tar.gz
+    cd ..
+
+Install the bundle  
+ `oc apply -f enmasse/install/bundles/enmasse`
```
Comment (ctron) [inline README.md]:
```suggestion

    oc apply -f enmasse/install/bundles/enmasse`
```

File: README.md [hunk 6]
```diff
@@ -87,6 +139,21 @@ And then apply the configuration:
...
 
     oc apply -f deploy/050-Workload
+    
+## See Results 
+
+Wait until the above steps are complete (i.e the simulator, producer, consumer, and web-console pods are running) 
+
+If everything is working correctly you should be able to open the web console at 
+   
+   `iot-simulator-console-iot-simulator.apps.<your cluster>`
```
Comment (ctron) [inline README.md]:
```suggestion
    iot-simulator-console-iot-simulator.apps.<your cluster>
```

**Review by ctron — CHANGES_REQUESTED** [review]
This looks great. Many thanks for the update. There are only a few nits. I made some suggestions, I hope they work for you.

**Comment by astoycos** [comment]
Looks good to me, I resolved all the changes!

**Review by ctron — APPROVED** [review]

**Comment by ctron** [comment]
Thank you again for the great work!

**Pull request merged**